site/docs/guides/components-basics.md

```diff
@@ -8,17 +8,17 @@ This guide walks you through the basics of how to style and work with components
 
 You use **Stylable** with a component file (for example using React), along with a **Stylable** CSS file that has the extention `.st.css`.
 
-> **Note**:
+> **Note**:  
 > This guide shows the JSX side of our [stylable-integration](https://github.com/wixplosives/stylable-integration) with React. 
 
-**Stylable** styles are similar to a type-system. Once you have declared that a CSS class is of the type `Button` for example, **Stylable** knows its internal structure and can match its internal parts and states.
+**Stylable** styles are similar to a type-system. Once you have declared that a CSS class is of the type `button` for example, **Stylable** knows its internal structure and can match its internal parts and states.
 
 Whether creating your own components or using components you imported from a 3rd party, you want to be able to access and style the internal parts of every component in the scope of your page or application. 
 
 
-## 1 Style a Component 
+## 1 Style a component 
 
-Let's say you have a `Button` component with a render function per this example. You can style its JSX using the `className` property.
+Let's say you have a `button` component with a render function per this example. You can style its JSX using the `className` property.
```

Comment:
capitalized.

Reply:
Fixed 
(Had debated this one cuz the render function has `<button> ..... </button> with only pseudo-elements between the render & it doesn't say "Button" like other jsx example).